dev/SearchBox.js

```diff
@@ -1,6 +1,9 @@
 import React, {Component, PropTypes as T} from 'react';
 import connect from '../src/connect.js';
 
-export default connect()(function({helper}) {
-  return <input onChange={(e) => helper.setQuery(e.target.value).search()} />;
+export default connect()(function({helper, search}) {
```

Comment:
Might lead to unexpected behavior. I like having a clear distinction between `helper.search` (sync) and `search` (debounced), but that does introduce a bit more API fragmentation.